airflow/models.py

```diff
@@ -122,44 +123,47 @@ class NullFernet(object):
     """
     is_encrypted = False
 
+    def __init__(self, k):
+        LoggingMixin().log.warn(
```

Comment:
Please add the logging mixin to the class itself: `class NullFernet(LoggingMixin):
`

Reply:
Fixed!